test/sagemaker_tests/pytorch/__init__.py

```diff
@@ -0,0 +1,182 @@
+# Copyright 2018-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+import os
+import sagemaker
+import boto3
+import subprocess
+import re
+import time
+from sagemaker import Session
+from base64 import b64decode
+
+
+def get_sagemaker_session(region):
+    return Session(boto_session=boto3.Session(region_name=region))
+
+
+def get_unique_name_from_tag(image_uri):
+    """
+    Return the unique from the image tag.
+
+    :param image_uri: ECR image URI
+    :return: unique name
+    """
+    return re.sub('[^A-Za-z0-9]+', '', image_uri)
+
+
+def get_account_id_from_image_uri(image_uri):
+    """
+    Find the account ID where the image is located
+
+    :param image_uri: <str> ECR image URI
+    :return: <str> AWS Account ID
+    """
+    return image_uri.split(".")[0]
+
+def get_region_from_image_uri(image_uri):
+    """
+    Find the region where the image is located
+
+    :param image_uri: <str> ECR image URI
+    :return: <str> AWS Region Name
+    """
+    region_pattern = r"(us(-gov)?|ap|ca|cn|eu|sa)-(central|(north|south)?(east|west)?)-\d+"
+    region_search = re.search(region_pattern, image_uri)
+    assert region_search, f"{image_uri} must have region that matches {region_pattern}"
+    return region_search.group()
+
+
+def ecr_repo_exists(ecr_client, repo_name, account_id=None):
+    """
+    :param ecr_client: boto3.Client for ECR
+    :param repo_name: str ECR Repository Name
+    :param account_id: str Account ID where repo is expected to exist
+    :return: bool True if repo exists, False if not
+    """
+    query = {"repositoryNames": [repo_name]}
+    if account_id:
+        query["registryId"] = account_id
+    try:
+        ecr_client.describe_repositories(**query)
+    except ecr_client.exceptions.RepositoryNotFoundException as e:
+        return False
+    return True
+
+def delete_file(file_path):
+    subprocess.check_output(f"rm -rf {file_path}", shell=True, executable="/bin/bash")
```

Comment:
nit: why not use invoke or directly use a python stdlib method?

Reply:
The function already exist [here](https://github.com/aws/deep-learning-containers/blob/master/test/sagemaker_tests/pytorch/training/utils/__init__.py#L60), moved a folder above. 
I will take this into consideration when i cleanup the [function](https://github.com/aws/deep-learning-containers/blob/master/test/sagemaker_tests/pytorch/training/utils/__init__.py#L60) in subsequent PR.